.github/workflows/build_images.yml

```diff
@@ -35,41 +31,62 @@ jobs:
           {name: 'aio-talk-recording', context: './Containers/talk-recording'},
           {name: 'aio-fulltextsearch', context: './Containers/fulltextsearch'},
           {name: 'aio-docker-socket-proxy', context: './Containers/docker-socket-proxy'},
-          {name: 'aio-whiteboard', context: './Containers/whiteboard'}]
-        exclude:
-          - platform: {arch: 'linux/arm64', tag: 'develop-arm64'}
-            container: {name: 'aio-clamav', context: './Containers/clamav'}
-
+          {name: 'aio-whiteboard', context: './Containers/whiteboard'}
+        ]
     steps:
       - name: Check out the repo
         uses: actions/checkout@v3
-
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v2
-
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v2
-
       - name: Login to DockerHub
         uses: docker/login-action@v2 
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
-      
-      - name: Docker metadata
-        id: metadata
-        uses: docker/metadata-action@v4
+      - name: Build docker image and push to dockerhub
+        uses: docker/build-push-action@v4
         with:
-          images: |
-            nextcloud/${{ matrix.container.name }}
-          tags: |
-            ${{ matrix.platform.tag }}
-
+          context: ${{ matrix.container.context }}
+          platforms: linux/amd64
+          no-cache: true
+          push: true
+          tags: nextcloud/${{ matrix.container.name }}:develop
+          
+  build-arm64:
+    runs-on: ubuntu-24.04-arm
```

Comment:
not sure if dependabot updates this, since there is currently no latest option possible

Reply:
I see, lets switch later when/if such an option is available :)